daemon/config/config_windows.go

```diff
@@ -76,6 +76,11 @@ func (conf *Config) IsRootless() bool {
 	return false
 }
 
+// IsCDIDisabled returns true because CDI is not supported on Windows.
+func (conf *Config) IsCDIDisabled() bool {
+	return true
+}
```

Comment:
What did we do before this; did we have special handling for Windows, or was having a non-empty list of CDI-dirs a no-op on Windows? Wondering - if we want to special-case windows - if we should have a validation step for that (then again, I don't think the API returns an error currently when trying to use CDI on a Windows daemon?)

Reply:
(Basically I was considering if we should have a generic `Features("foo")` - or similar - method to check whether a feature is enabled. To prevent having to implement a dedicated accessor for every feature (as we tried to keep the `Features` field somewhat generic 🤔

Reply:
On our side it would be a no-op - not sure what would Buildkit do though.

Regarding feature flag - I don't think we will actually want to keep it long term? Just leave it as an escape hatch initially, but remove in next release if there are no issues?

Reply:
Yeah, maybe there's still some use for it, for situations where an admin wants to make sure it's not available.

That said; if it's gonna be temporary, that's probably even a stronger motivation to not add a new Method specifically for it? 🙈 😅

Reply:
Right, turned into a private helper function instead of making it a publicly exported method of Config.